dev-util/web_page_replay_go/web_page_replay_go-20210916.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit go-module
+
+EGIT_COMMIT=57c353ca10632889d961e58f8acc5ca671c2c17b
```

Comment:
EGIT_COMMIT is fine but I think it's a bit misleading because git-r3 uses that variable.